api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -39,14 +40,15 @@ public NotebooksServiceImpl(Provider<ClusterApi> clusterApiProvider,
   }
 
   private ClusterRequest createFirecloudClusterRequest(String userEmail) {
-    ClusterRequest firecloudClusterRequest = new ClusterRequest();
     Map<String, String> labels = new HashMap<String, String>();
     labels.put(CLUSTER_LABEL_AOU, "true");
     labels.put(CLUSTER_LABEL_CREATED_BY, userEmail);
-    firecloudClusterRequest.setLabels(labels);
-    firecloudClusterRequest.setJupyterUserScriptUri(
-        workbenchConfigProvider.get().firecloud.jupyterUserScriptUri);
-    return firecloudClusterRequest;
+    return new ClusterRequest()
+        .labels(labels)
+        .jupyterUserScriptUri(workbenchConfigProvider.get().firecloud.jupyterUserScriptUri)
+        .machineConfig(new MachineConfig()
+            .masterDiskSize(100 /* GB */)
```

Comment:
We could FWIW make these configurable in config as well. That would let us change this up in different environments if we found it useful.

Reply:
ACK - In the interest of minimizing unused flags, I think my preference is to keep as constants until we decide we want to do that (or if decide we need the ability to quickly change this config in prod).